plugins/tableresize/plugin.js

```diff
@@ -90,6 +92,17 @@
 
 				pillarWidth = Math.max( pillarRight - pillarLeft, 3 );
 
+				// In case of IE and collapsed table border, we must substract all borders
+				// from previous and current cells (#2823).
+				if ( isIE && isBorderCollapse ) {
+					var borderLeft = getBorderWidth( table, 'left' ),
+						borderRight = getBorderWidth( table, 'right' ),
+						index = td.$.cellIndex + 1;
+
+					pillarLeft -= ( borderLeft * index ) + ( borderRight * index ) + pillarWidth;
```

Comment:
Shouldn't we recalculate `pillarWidth` after adjusting `pillarLeft` here? like in line 93 above `pillarWidth = Math.max( pillarRight - pillarLeft, 3 );`? I see it makes a pillar little wider (which makes it easier to "grab").